### ISSUE TITLE
linux://python/ray/tests:test_autoscaler is flaky

### ISSUE THREAD

**Issue by can-anyscale** [ray-project/ray#41186]
### What happened + What you expected to happen

https://flakey-tests.ray.io/?owner=serverless

It is also marked as flaky in https://github.com/ray-project/ray/blob/master/ci/ray_ci/serverless.tests.yml#L2

### Versions / Dependencies

master

### Reproduction script

https://github.com/ray-project/ray/blob/master/ci/ray_ci/serverless.tests.yml#L2

### Issue Severity

Medium: It is a significant difficulty but I can work around it.

**Comment by anyscalesam** [comment]
This one is pretty bad last 2w - we need to root cause into the test failure or decide if this test isn't important.

**Comment by anyscalesam** [comment]
Still poor - @rkooo567 should we make this a release-blocker and fix this week?

**Comment by rkooo567** [comment]
yes we should fix it

**Comment by rkooo567** [comment]
this doesn't belong to us now. @rickyyx do we own the autoscaler v1?

**Comment by rickyyx** [comment]
I thought we are owning `serverless` CI tests? Didn't you change the tag?

**Comment by rkooo567** [comment]
I didn't move this because I thought we only own autoscaler v2.

I can move it to the core team's task

**Comment by rickyyx** [comment]
yeah - I guess I prob have the most context of v1 now.

**Comment by rkooo567** [comment]
I will assign this to you for now (but feel free to find a person delegate it!)

**Comment by rkooo567** [comment]
https://github.com/ray-project/ray/pull/41484

**Comment by anyscalesam** [comment]
@rickyyx can you confirm the way we validate this is we are going to configure the new autoscaler v2 path for this test and confirm it passes? if passes than we can close this one as well?

**Comment by rickyyx** [comment]
> @rickyyx can you confirm the way we validate this is we are going to configure the new autoscaler v2 path for this test and confirm it passes? if passes than we can close this one as well?

yeah - it should be eventually replaced by the v2 autoscaler tests.

**Comment by rkooo567** [comment]
duplicate https://github.com/ray-project/ray/issues/42624
